beginner_setup_guide.md

```diff
@@ -0,0 +1,164 @@
+* [arduino-esp8266fs-plugin](https://github.com/esp8266/arduino-esp8266fs-plugin/releases/)
+* ESP8266 Microcontroller  
+* Breakout Board
+* GB Link Cable
+* 3 Jumper Cables
+
+### Optional
+* Soldering iron
+* Perf-Board
+* [OLED Screen](https://www.amazon.de/gp/product/B07BDFXFRK)
+
+## Step 1: Flash the board
+
+### 1.1 Install the Arduino IDE 
+Download the Arduino IDE either from the Website (Windows/MacOS) from [here](https://www.arduino.cc/en/Main/Software) and follow the basic installation process.
+ It is recommended for Linux Users to use your packet manager to install the IDE. 
+
+### 1.2 Install ESP8266 in Arduino
+Start the Arduino IDE and navigate to ```File -> Preferences``` and add the following line to "Additional Boards Manager URLs:"
+
+````
+http://arduino.esp8266.com/stable/package_esp8266com_index.json
+````
+Confirm with "OK" and navigate to ```Tools -> Board -> Board Manager```. In the filter mask search for ```esp8266``` and install the package. 
+
+Lastly navigate to ```Tools -> Board``` and select your EPS8266 module
+
+### 1.3 Install arduino-esp8266fs-plugin
+
+Download the latest version of the ```arduino-esp8266fs-plugin``` from [here](https://github.com/esp8266/arduino-esp8266fs-plugin/releases/)
+
+In the Arduino IDE navigate to ```File -> Preferences``` and check for your ```Sketchbook location``` path. Navigate to that path in your file browser. If needed create the folder ```tools``` and extract the ```esp8266fs.jar``` into that folder.
+
+Restart the Arduino IDE after that.
+
+### 1.4 Download the websocket-gbp-emulator github code and additional data
+
+Download the full code from [here](https://github.com/HerrZatacke/websocket-gbp-emulator/archive/master.zip) as a ZIP file and extract it to your home directory. In your file browser navigate into the extracted folder and then into the ```websocket-gbp-emulator``` subfolder. Here create a copy of the ```config.h.sample.txt```. Rename the new file to ```config.h``` 
+
+It is recommended to download the additional data for the GB-Printer-Web interface. To automatically download the additional data do:
+
+#### Windows
+Go to your Start Menu, type ```powershell``` and navigate to the directory you just extracted (e.g. cd ```YourUsername\Downloads\websocket-gbp-emulator\```)
+Run the download script with ```.\update_w.ps1```
+
+#### MacOS / Linux
+Open your Terminal and navigate to the files you just extracted (e.g. ```cd ~/Downloads/websocket-gbp-emulator/```)
+Type ```chmod +x update_w.sh```
+Type ```./update_w.sh```
+
+### 1.5 Upload the code and data
+
+Connect your ESP8266 Board via USB to your PC. Navigate to ```File -> Open ...``` navigate to the folder you unzipped in the previous step. Open the ```websocket-gbp-emulator.ino``` 
+
+To confirm that the code was downloaded correctly navigate to ```Sketch -> Verify/Compile```. If everything works go to ```Sketch -> Upload``` and wait until the upload is finished.  
```

Comment:
Maybe also add info on selecting the board/port etc

Reply:
Board selection is addressed in Line 39, will ad Port Selection and adding the library. :)